sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_prebuilts_from_url_async.py

```diff
@@ -236,16 +236,17 @@ async def test_receipt_bad_url(self, client):
 
     @FormRecognizerPreparer()
     @DocumentAnalysisClientPreparer()
-    @recorded_by_proxy_async
-    async def test_receipt_url_pass_stream(self, client):
+    async def test_receipt_url_pass_stream(self, **kwargs):
+        client = kwargs.get("client", None)
         
         with open(self.receipt_png, "rb") as fd:
             receipt = fd.read(4)  # makes the recording smaller
 
-        with pytest.raises(HttpResponseError):
+        with pytest.raises(ValueError) as e:
             async with client:
                 poller = await client.begin_analyze_document_from_url("prebuilt-receipt", receipt)
                 result = await poller.result()
+                assert str(e) == "'document_url' needs to be of type 'str'. Please see `begin_analyze_document()` to pass a stream."
```

Comment:
Does this pass? the error message in the client says "byte stream" and the string we're testing here just says "stream". I think the indentation for the `assert str(e) == ...` might need to be adjusted?

Reply:
Hmm let me check...it did pass when I ran it and the ci tests are passing...I'll check the messages and the indentation

Reply:
I think it might be throwing on L247 and not actually executing this assertion line

Reply:
Thanks for catching that! Yes the indentation was not letting us get to the check. Also the comparison was a bit off...pushing the updates soon!